newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -521,6 +521,23 @@ const PrivateAssetPackInformationPage = ({
     { subscription, privateAssetPackListingData, isAlreadyReceived }
   );
 
+  const smartObjectsCount = React.useMemo(
+    () => {
+      if (!assetPack) {
+        return 0;
+      }
+      let smartObjectsCount = 0;
+      for (const type in assetPack.content) {
+        const assetCount = assetPack.content[type];
+        if (!sortedContentType.includes(type)) {
```

Comment:
It feels a bit light as a check to decide if the object is a smart object. If we add 3D particle emitters or 3D cubes in a pack, it will display them as a smart objects.

Reply:
If smart objects mean an object with some logic, it's the case for any object so I think it's not a big deal.
We can add lines for more objects in the future if we think it becomes useful.

Reply:
But it's quite deceptive when users purchase something and they end up with boring objects.

Reply:
There is no easy way to know if an object comes from engine or an event-based extension. And even in this case, the limit is blurry because the 3D particle emitter is from an event-based extension and it may not fit everyone definition of "smart object".
I think when there are new cases, we can add specific lines for them.
The choice of words:
- "smart object" allow to avoid diminishing the appeal for asset packs with custom objects (which is an actual use-case)
- "other" avoid the risk of mislabeling new objects (which is only a bug that can be fixed easily)

Reply:
Smart objects is probably fine. If some are becoming super "well known", we'll add specific names for them in the future.